docs/eggs/installing-a-script-as-an-executable.md

```diff
@@ -6,4 +6,7 @@ To install a module, simply replace *deno* with *eggs*.
 
 ```shell script
 eggs install --allow-read --allow-run --allow-write --allow-net --unstable https://x.nest.land/denon@2.2.0/denon.ts
-```
\ No newline at end of file
+```
+
+<!-- TODO(@oganexon): update screenshot -->
```

Comment:
Is this still a todo? I think the screenshot looks fine.

Reply:
Is user-images.githubusercontent.com a safe place to host images ?

Reply:
Fair enough. We can fix this later